package.json

```diff
@@ -20,17 +21,23 @@
   "devDependencies": {
     "@types/jest": "^24.0.22",
     "@types/node": "^12.12.6",
+    "@typescript-eslint/eslint-plugin": "^2.6.1",
+    "@typescript-eslint/parser": "^2.6.1",
+    "eslint": "^6.6.0",
+    "eslint-config-prettier": "^6.5.0",
+    "eslint-plugin-prettier": "^3.1.1",
     "jest": "^24.9.0",
     "mock-jwks": "^0.2.0",
-    "nock": "^11.7.0",
```

Comment:
My library expects a peer dep of `nock@10`, not `nock@11`. While this was not the issue it is still good to use the correct peer dep. You get a warning from npm when you install the deps about this.